debian/linuxcnc-doc-de.docs

```diff
@@ -0,0 +1,6 @@
+usr/share/doc/linuxcnc/LinuxCNC_Developer_de.pdf
+usr/share/doc/linuxcnc/LinuxCNC_Documentation_de.pdf
+usr/share/doc/linuxcnc/LinuxCNC_Getting_Started_de.pdf
+usr/share/doc/linuxcnc/LinuxCNC_Integrator_de.pdf
+
+docs/html/gcode.html
```

Comment:
I believe this should be docs/html/de/gcode.html.  Same for the nb path.

Reply:
Oops, good catch.